examples/pub-sub-rewind/javascript/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+    <link rel="stylesheet" href="src/styles.css" />
+    <title>Pub/Sub rewind channel</title>
+  </head>
+  <body class="font-inter">
+    <div id="landing-page" class="min-h-screen flex items-center justify-center bg-gray-100">
+      <div class="bg-white p-8 rounded-lg shadow-lg w-96">
+        <h2 class="text-2xl mb-6 text-center">Live Football League Odds</h2>
+        <p class="text-sm text-center">Watch real-time odds movement for today's Football League match.</p>
```

Comment:
```suggestion
        <p class="text-sm text-center">Simulate 10 betting odds for a sports game before attaching.</p>
```

Maybe something like this to show what's happening?